types/schemas/postSubmissionApplication/Metadata.ts

```diff
@@ -0,0 +1,9 @@
+import {DateTime} from '../../shared/utils';
+import {PrototypePlanXMetadata} from '../prototypeApplication/Metadata';
+
+export interface PostSubmissionMetadata extends PrototypePlanXMetadata {
+  /**
+   * This is the date this application was published to the public
+   */
+  publishedAt: DateTime;
+}
```

Comment:
Metadata is an interesting one to think about !

Our original thinking of having "PlanXMetadata", rather than "ApplicationMetadata", was that the metadata is specific to how the application was completed _within PlanX as a system_. If an application were to come from another system, it have similar and different aspects than what PlanX gathers - so it would be like `AnotherSystemMetadata`. 

I wonder if `Metadata` for `PostSubmission` at this stage should actually be `BOPSMetadata`?

Reply:
PublishedAt is a funny one. I originally created `publishedApplicationSchema` and then I switched it round to be the `PostSubmissionSchema` so we could do the redacted versions later on.

PublishedAt is the date it was published to the public API in the `publishedApplicationSchema` but it could equally apply to the private API in the `PostSubmissionSchema`

We need a publishedAt date in the dpr for now and its not really a 'stage' in the same sense as the others but maybe we could move it to `data.application` ? I've updated the metadata to include the BaseMeta data as well so they both allow for PublishedAt.

![image](https://github.com/user-attachments/assets/12018cb5-2119-4db0-94ba-26dc0d74741d)